MetaMask/metamask-extension
# UX: Add 'Add' icon to Add Network button

## **Description**

Design wanted to add the "+" to the Add Network button

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1941

## **Manual testing steps**

1. Open the network menu
2. See the "+" icon on the button

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

<img width="426" alt="SCR-20240122-nusi" src="https://github.com/MetaMask/metamask-extension/assets/46655/53661e71-a383-44c8-b73c-a1b7e5400026">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
